src/fastoad/gui/mission_viewer.py

```diff
@@ -58,69 +58,56 @@ def add_mission(self, mission_data: Union[str, pd.DataFrame], name=None):
         else:
             raise TypeError("Unknown type for mission data, please use .csv of DataFrame")
 
-        # Initialize widgets when first mission is added
-        if len(self.missions) == 1:
-            self._initialize_widgets()
-
-    def _initialize_widgets(self):
-        """
-        Initializes the widgets for selecting x and y
-        """
+    def display(self):
```

Comment:
Though I am not sure if it was useful, the previous display method was returning the DisplayHandle instance. It costs nothing to keep the same behavior.

Also, we miss a docstring here.

Reply:
Fixed in 95306126